crates/nu-command/src/filters/par_each.rs

```diff
@@ -146,16 +148,15 @@ impl Command for ParEach {
                                 redirect_stdout,
                                 redirect_stderr,
                             ) {
-                                Ok(v) => v,
+                                Ok(v) => v.into_value(span),
```

Comment:
This will materialize a `PipelineData::ListStream` to a `Value::List` but I think there is no way we can keep this a lazy iterator anyways when we run the evaluation in the parallel threads and collect afterwards.

Reply:
Seems to me like evaluating the streams in parallel makes sense, but I don't know enough about the codebase to make a meaningful comment here about whether it is possible or desirable to keep them lazy.

Reply:
@rgwood is probably our resident streaming expert at the moment but there should be no harm to this change.